packages/jsii-pacmak/lib/targets/java.ts

```diff
@@ -754,20 +754,20 @@ class JavaGenerator extends Generator {
         for (const prop of props) {
             if (prop.optional) { this.code.line(JSR305_NULLABLE); }
             // tslint:disable-next-line:max-line-length
-            this.code.line(`private${prop.immutable ? ' final' : ''} ${prop.fieldJavaType} ${prop.fieldName} = ${_validateIfNonOptional(`_${prop.fieldName}`, prop)};`);
+            this.code.line(`private${prop.immutable ? ' final' : ''} ${prop.fieldJavaType} $${prop.fieldName} = ${_validateIfNonOptional(`_${prop.fieldName}`, prop)};`);
```

Comment:
I think `fieldName` should include the "$" prefix so you don't have to repeat this.
Also, I would use an underscore, which I believe is more idiomatic, no?